mantatail.py

```diff
@@ -200,6 +276,10 @@ def queue_quit_message_for_other_users(self) -> None:
             receiver.send_que.put((message, self.get_user_mask()))
 
     def send_string_to_client(self, message: str, prefix: Optional[str]) -> None:
+        """
+        Formats a message taken from the user's send queue, converts it to
+        bytes (encoded with latin-1) and sends it to the client.
+        """
```

Comment:
While this explains exactly what the method does, it isn't very useful. Someone working on Mantatail will likely find this, think that they are supposed to use it for everything, and never learn about the send queue.

```suggestion
        """
        Send a string to the client, without using the send queue.

        In most cases, you should put a message to the send queue instead of using this method.
        """
```